app/models/miq_request/state_machine.rb

```diff
@@ -0,0 +1,36 @@
+module MiqRequest::StateMachine
```

Comment:
This looks copied&pasted from `Job::StateMachine`...should we just make this a mixin that both can use?

Reply:
Yes, I think we should